dpx/src/dpx_pdfdev.rs

```diff
@@ -245,18 +284,18 @@ pub struct TextState {
     pub ref_y: spt_t,
     pub raise: spt_t,
     pub leading: spt_t,
-    pub matrix: C2RustUnnamed_2,
+    pub matrix: Matrix,
     pub bold_param: f64,
     pub dir_mode: i32,
     pub force_reset: i32,
     pub is_mb: i32,
 }
 #[derive(Copy, Clone)]
 #[repr(C)]
-pub struct C2RustUnnamed_2 {
+pub struct Matrix {
```

Comment:
According to https://github.com/tectonic-typesetting/tectonic/blob/87f90c7a7c2095fcf469b4db124117f79b231660/tectonic/dpx-pdfdev.c#L380 let's call it font matrix to make it clearer.

Reply:
Done.